merlin/data/dataorganization.py

```diff
@@ -112,9 +112,10 @@ def get_data_channel_index(self, dataChannelName: str) -> int:
             # TODO this should raise a meaningful exception if the data channel
             # is not found
         """
-        return self.data[self.data['channelName'].apply(
-            lambda x: str(x).lower()).str.match(
-            dataChannelName.lower())].index.values.tolist()[0]
+        return self.data[self.data['channelName'] == dataChannelName].index.values.tolist()[0]
```

Comment:
Why is this changed? It seems it could lose backward compatibility by requiring the names to match case.

Reply:
I think it's reasonable to remove the case sensitivity in the update as you want. The reason this was changed was because in instances where you had two names that contained the same string (i.e. 'ABCD' and 'ABCDE'), depending on the order they appeared in in the file, it could be impossible to get it to generate a mosaic for the one you wanted to target since it just took the first element of the match previously.